### ISSUE TITLE
Report overflow

### ISSUE THREAD

**Issue by TuomLarsen** [secnot/rectpack#7]
It would be very useful to somehow report whether all the rectangles were fitted into available bins or not (there were some rectangles left out).

**Comment by secnot** [comment]
You can obtain the number of rectagles packed with:

```python
packer.pack()

rectangles_packed = sum(len(bin) for bin in packer)
```
and then compare with the rectangles added. 

But perhaps a **packed_count()** method or something similar could be more convenient.

Thank you for the suggestion

**Comment by TuomLarsen** [comment]
And thank you for the answer!

**Comment by kg-2** [comment]
I agree that it should be more explicit when a rect isn't able to be added- that would be the expected behavior instead of silently just ignoring it

**Comment by secnot** [comment]
For many problems there are more rectangles than space available, so it makes no sense to return the rectangles not packed several times until they are placed in a bin
 
If you need to know which rectangles where packed the simplest solution is to provide an id for each one and then compare the results with a dictionary containing the ids.
